apps/meteor/server/models/raw/LivechatVisitors.ts

```diff
@@ -49,6 +49,16 @@ export class LivechatVisitorsRaw extends BaseRaw<ILivechatVisitor> implements IL
 		return this.findOne(query);
 	}
 
+	findOneVisitorByPhoneAndSource(phone: string, sourceFilter: Filter<ILivechatVisitor>): Promise<ILivechatVisitor | null> {
```

Comment:
Should these accept `options`? I know you're not using them but that would allow us to use this with projections if we need it in the future without having to change much things, wdyt?